pusher/util.py

```diff
@@ -23,6 +23,11 @@
     text = 'a string'
 
 
+if sys.version_info < (3,):
+    byte_type = 'a python2 str'
```

Comment:
I think this should be `'a python2 bytes or str'` since in Python 2 they refer to the same underlying bytes type, according to https://cython.readthedocs.io/en/latest/src/tutorial/strings.html

Reply:
Actually, forget this. `bytes` is just an alias fro `str`. I think it's ok to just mention the canonical type.